sap-schemas/v3.0/schema.json

```
@@ -718,6 +718,46 @@
           "type": "boolean",
           "default": false,
           "description": "Indicates if the marked object represents a root entity. A root entity is the root of a business object. It has a distinct existence. Examples are real-world objects, like a workforce person or a piece of equipment or other more abstract real-world concepts, like a contract or an order. It has an independent lifetime / identity and is globally and uniquely addressable (via the unique identifier provided by the property referenced in x-sap-odm-oid field)."
+        },
+        "x-sap-dpp-entitySemantics": {
+          "type": "string",
+          "description": "Primary meaning of the entities in the annotated entity set. Entities annotated with x-sap-dpp-entitySemantics are synonymous to x-sap-dpp-isPotentiallyPersonal.",
+          "enum": [
+            "sap:DataSubject",
+            "sap:DataSubjectDetails",
+            "sap:Other"
+           ]
+        },
+        "x-sap-dpp-dataSubjectRole": {
+          "type": "string",
+          "description": "Role of the data subjects in this set (e.g. employee, customer). Values are application-specific. Can be a static value or a Path expression If the role varies per entity"
```

```suggestion
          "description": "Role of the data subjects in this set (e.g. employee, customer). Values are application-specific."
```
OpenAPI does not allow Path expressions, we would have to define how these are expressed if they are needed.